pkgs/applications/misc/homely/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, pythonPackages }:
+
+pythonPackages.buildPythonApplication rec {
+  version = "0.15.3";
+  pname = "homely";
+
+  src = pythonPackages.fetchPypi {
+    inherit pname version;
+    sha256 = "2aa7461b583660bf8d9ad776c184fa7a1272141a5c0faaf78a76bc3e8c39db6d";
+  };
+
+  buildInputs = with pythonPackages; [ attrs setuptools_scm ];
```

Comment:
Is `attrs` required? I tried removing it (with Mic92's suggestion) and the package built fine.